addon/components/hyper-table-v2/cell.ts

```diff
@@ -1,5 +1,37 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import TableHandler from '@upfluence/hypertable/core/handler';
+import { Column, ResolvedRenderingComponent, Row } from '@upfluence/hypertable/core/interfaces';
 
-interface HyperTableV2CellArgs {}
+interface HyperTableV2CellArgs {
+  handler: TableHandler;
+  column: Column;
+  row: Row;
+  loading: boolean;
+}
 
-export default class HyperTableV2Cell extends Component<HyperTableV2CellArgs> {}
+export default class HyperTableV2Cell extends Component<HyperTableV2CellArgs> {
+  @tracked loadingCellComponent: boolean = false;
+  @tracked cellComponent?: ResolvedRenderingComponent;
+
+  constructor(owner: unknown, args: HyperTableV2CellArgs) {
+    super(owner, args);
+
+    this.loadingCellComponent = true;
+
+    if (!args.loading) {
```

Comment:
It is possible that if `args.loading = true`, the `loadingCellComponent` stay true of ever because it never goes in `finally` function?

Reply:
`@loading` is passed from the parent template and is related to the rows being fetched here, and it's set to `false` whatever happens in the finally (of the table handler)